sparkles/tests/test_review.py

```diff
@@ -182,3 +184,136 @@ def test_run_aca_review_function():
         {'text': 'P2: 2.84 less than 3.0 for ER', 'category': 'critical'},
         {'text': 'ER count of 9th (8.9 for -9.9C) mag guide stars 1.91 < 3.0',
          'category': 'critical'}]
+
+
+def test_roll_outside_range():
+    """
+    Run a test on obsid 48334 from ~MAR1119 that is at a pitch that has 0 roll_dev
+    and is at a roll that is not exactly nominal roll for the attitude and date.
+    The 'att' ends up with roll outside of the internally-computed roll_min / roll_max
+    which caused indexing excption in the roll-options code.  Fixed by PR #91 which
+    includes code  to expand the roll_min / roll_max range to always include the roll
+    of the originally supplied attitude.
+    """
+    kw = {'att': [-0.82389459, -0.1248412, 0.35722113, 0.42190692],
+          'date': '2019:073:21:55:30.000',
+          'detector': 'ACIS-S',
+          'dither_acq': (7.9992, 7.9992),
+          'dither_guide': (7.9992, 7.9992),
+          'focus_offset': 0.0,
+          'man_angle': 122.97035882921071,
+          'n_acq': 8,
+          'n_fid': 0,
+          'n_guide': 8,
+          'obsid': 48334.0,
+          'sim_offset': 0.0,
+          't_ccd_acq': -10.257559323423214,
+          't_ccd_guide': -10.25810835536192}
+    aca = get_aca_catalog(**kw)
+    acar = aca.get_review_table()
+    acar.get_roll_options()
+    assert Quat(kw['att']).roll <= acar.roll_info['roll_max']
+    assert Quat(kw['att']).roll >= acar.roll_info['roll_min']
```

Comment:
I left this one not using acar.att.roll so that we are explicitly testing what we care about... that the requested roll in the kwargs is in the returned roll range in roll_info.